modules/__tests__/enhancer-test.js

```diff
@@ -155,14 +155,19 @@ describe('Enhancer', () => {
     });
   });
 
-  it('manually populates static properties for IE <10', () => {
-    class Composed extends Component { }
+  it('manually populates all static properties for IE <10', () => {
+    class Composed extends Component {
+      static staticMethod () {
+        return { bar: 'foo' };
+      }
+    }
 
     Composed.defaultProps = { foo: 'bar' };
```

Comment:
Since we're using babel anyway, move defaultProps to the class:

```as
class Composed extends Component {
  static staticMethod () {
    return { bar: 'foo' };
  }

  defaultProps = {foo: 'bar'}
}